packages/playground/src/components/select_vm_image.vue

```diff
@@ -19,11 +24,15 @@
       :value="flist"
       #="{ props }"
     >
-      <v-text-field label="Flist" v-model="flist" v-bind="props" />
+      <input-tooltip tooltip="Add a custom flist link, you can vist our tf-hub for more information.">
```

Comment:
maybe add the link to the hub
```suggestion
      <input-tooltip tooltip="Add a custom flist link, you can visit our 0-hub for more information.">
```